src/app/lib/utilities/langHandler/index.test.js

```diff
@@ -1,18 +1,27 @@
 import getLangOverride from '.';
 
 describe('getLangOverride', () => {
-  it('should return variant if pathname is correct', () => {
+  it('should return the name of a service lang override for the given service and page lang combination', () => {
     expect(
-      getLangOverride({ pathname: '/ukrainian/ukraine_in_russian' }),
-    ).toEqual('ru-UA');
-    expect(
-      getLangOverride({ pathname: '/ukrainian/ukraine_in_russian.amp' }),
+      getLangOverride({
+        service: 'ukrainian',
+        pageLang: 'ru',
+      }),
     ).toEqual('ru-UA');
   });
 
-  it('should return null if pathname is incorrect', () => {
-    expect(getLangOverride({ pathname: '/ukrainian/not_russian' })).toEqual(
-      null,
-    );
+  it('should return undefined for the given service and page lang combination', () => {
+    expect(
+      getLangOverride({
+        service: 'russian',
+        pageLang: 'ru',
+      }),
+    ).toEqual(undefined);
+    expect(
+      getLangOverride({
+        service: 'russian',
+        pageLang: undefined,
+      }),
+    ).toEqual(undefined);
```

Comment:
Could we use a service other than Russian here, I think this test is intended to check the change you have made is non-breaking; using Russian gives the impression you are testing behaviour specific to the Russian service given the context of this PR? Maybe use Mundo or something to make that clear?

Reply:
That's right, yeah.
Ok, thanks for the suggestion. I've amended this test to check a service other than Russian.